aiida_quantumespresso/parsers/projwfc.py

```diff
@@ -366,6 +344,14 @@ def parse(self, **kwargs):
         Dos_out.set_y(dos, 'Dos', 'states/eV')
         self.out('Dos', Dos_out)
 
+    def parse_stdout(self, stdout, out_info_dict):
+        """Parse the ``stdout`` content of a Quantum ESPRESSO ``projwfc.x`` calculation.
+
+        :param stdout: the stdout content as a string.
+        """
+        out_info_dict['out_file'] = stdout.split('\n')
```

Comment:
I know using the `out_info_dict` is messy as hell, but that'll be fixed in https://github.com/aiidateam/aiida-quantumespresso/pull/749. :)